packages/webpack-build-scripts/externals.js

```diff
@@ -16,11 +16,14 @@ module.exports = externalize({
     "jquery": "$",
     "moment": "moment",
     "moment-timezone": "moment",
+    "popper.js": "Popper",
     "react": "React",
-    "react-transition-group": "ReactTransitionGroup",
     "react-day-picker": "DayPicker",
     "react-dom": "ReactDOM",
-    "tslib": "tslib",
+    "react-popper": "ReactPopper",
+    "react-transition-group": "ReactTransitionGroup",
+    "resize-observer-polyfill": "ResizeObserver",
+    "tslib": "window",
```

Comment:
i think _this_ is the line that actually fixes #3262

Reply:
how does this work? I see very little documentation about this "window" option

Reply:
i had to dig through the tslib code to figure out what global it exposes. turns out it adds methods directly to `window`. same as dom4.